src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -1587,12 +1587,19 @@ private void Abort(Exception abortException)
                 (_httpStreams.Count == 0) &&
                 ((nowTicks - _idleSinceTickCount) > connectionIdleTimeout.TotalMilliseconds))
```

Comment:
45990[**45899!
```suggestion
                 ((nowTicks - _idleSinceTickCount) > connectionIdleTimeout.TotalMilliseconds)))?7&
```
**]()